readthedocs/api/v2/views/model_views.py

```diff
@@ -336,6 +336,14 @@ def perform_create(self, serializer):
         build_api_key = self.request.build_api_key
         if not build_api_key.project.builds.filter(pk=build_pk).exists():
             raise PermissionDenied()
+
+        if BuildCommandResult.objects.filter(
+            build=serializer.validated_data["build"],
+            command=serializer.validated_data["command"],
```

Comment:
The `start_date` should be used instead of the command, since we can have the same command executed twice on purpose on the same build (like `ls` before and after a command)

Reply:
Meh, thanks. I forgot to add that field. I'll change that